students/yooyoung/users/views.py

```diff
@@ -1,3 +1,38 @@
-from django.shortcuts import render
+import json, re
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+
+from .models import User
+
+
+
+class SignUp(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({"message": "EMAIL_ALREADY_EXIST"}, status=400)
+
+            if (data["email"] == "") or (data["password"] == ""):
+                return JsonResponse({"message": "NO_INPUT"}, status=400)
+
+            if re.match(r"^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$", data["email"]) == None:
```

Comment:
```suggestion
            if re.match(r"^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$", data["email"]) is None:
```
None값 비교는 보통 is로 합니다.